src/Streamly/Tutorial.hs

```diff
@@ -1165,18 +1244,13 @@ import Control.Monad.Trans.Class   (MonadTrans (lift))
 -- specify the type of the stream as 'WAsync'.
 --
 -- @
--- import qualified "Streamly.Prelude" as S
 --
--- main = S.'drain' . S.'fromWAsync' $ do
---     x <- S.'fromFoldable' [1,2]
---     y <- S.'fromFoldable' [3,4]
---     S.'fromEffect' $ putStrLn $ show (x, y)
--- @
--- @
--- (1,3)
--- (2,3)
--- (1,4)
--- (2,4)
+-- Stream.drain . Stream.fromWAsync $ do
+--     x <- Stream.fromFoldable [1,2]
+--     y <- Stream.fromFoldable [3,4]
+--     Stream.fromEffect $ putStrLn $ show (x, y)
+--
+-- print (1,3) (2,3) (1,4) (2,4) in any order
```

Comment:
We can apply 1 sec delay using`Stream.delay` on each stream and then the output should be predictable.

Reply:
Tried with delay still output is not predictable:
 
Stream.drain . Stream.fromWAsync $ do
     x <- Stream.delay 1 $ Stream.fromFoldable [1,2]
     y <- Stream.delay 1 $ Stream.fromFoldable [3,4]
     Stream.fromEffect $ putStrLn $ show (x, y)
:}'
expected: (1,3) 
          (2,3) 
          (1,4) 
          (2,4) 
 but got: (1,3)
          (1,4)
           ^
          (2,3)
          (2,4)